### ISSUE TITLE
Did progress stop?

### ISSUE THREAD

**Issue by BaseMax** [devWaz/30-Days-of-JavaScript#1]
I know maybe you are busy.
Anyway, it's really awesome and cool repository.

Best wishes to you.
^_^

**Comment by devWaz** [comment]
Yes, i've been busy. i plan on continuing soon tho. Thanks for reaching out 👍🏾
